### PR TITLE
[NUI] Add WatchView

### PR DESCRIPTION
**Pull request by tscholb**
### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/WatchView.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+        /// <summary>
+        /// Pauses a given Watch.
+        /// </summary>
+        /// <returns>True on success, false otherwise.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool PauseWatch()
+        {
+            bool ret = Interop.WatchView.WatchView_PauseWatch(swigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
```
Comment (m-macias) [inline src/Tizen.NUI/src/public/WatchView.cs:57]: Please don't write the if-statement in the single line. I am pretty sure that is against used convention and will be reported by one of our tools.

File: src/Tizen.NUI/src/public/WatchView.cs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    using System;
```
Comment (m-macias) [inline src/Tizen.NUI/src/public/WatchView.cs:21]: Was it intentional that you have declared using statement inside and outside of the namespace?

File: src/Tizen.NUI/src/public/WatchView.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+        internal WatchView Assign(WatchView handle)
+        {
+            WatchView ret = new WatchView(Interop.WatchView.WatchView_Assign(swigCPtr, WatchView.getCPtr(handle)), false);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        /// <summary>
+        /// To make the Button instance be disposed.
```
Comment (m-macias) [inline src/Tizen.NUI/src/public/WatchView.cs:100]: It may be grammatically correct, but I would suggest a change.

**Comment by rabbitfor** [comment]
NUI patch needs prefix "[NUI]" for the commit message :)

**Comment by hyunjushin** [comment]
it's not public feature yet. please hide APIs.

**Comment by tscholb** [comment]
I close this PR due to duplicate PR #1960

**Pull request closed without merging**